libs/react-helpers/src/lib/assets.ts

```diff
@@ -16,14 +16,15 @@ export interface Asset {
   source: ERC20AssetSource | BuiltinAssetSource;
 }
 
-interface AssetEdge {
-  __typename: 'AssetEdge';
-  node: Asset;
+export enum AssetStatus {
+  STATUS_ENABLED = 'STATUS_ENABLED',
+  STATUS_PENDING_LISTING = 'STATUS_PENDING_LISTING',
+  STATUS_PROPOSED = 'STATUS_PROPOSED',
+  STATUS_REJECTED = 'STATUS_REJECTED',
 }
 
-interface AssetsConnection {
-  __typename: 'AssetsConnection';
-  edges: (AssetEdge | null)[] | null;
+export interface AssetWithStatus extends Asset {
+  status: AssetStatus;
```

Comment:
thought (non-blocking): wondering if we should just be using a basic interface here with status. The asset type requires symbol, decimals ect. none of which are used in `getEnabledAssets` so not sure we need to be extending that for a function that does not require those fields to work.

Reply:
True we do need the "full" type there but because it returns `Asset[]` the conversion from just `{ state: AssetStatus }[]` to `Asset[]` requires middle step of converting it to `unknown` first which imo looks bad (`as unknown as Asset[]`)